tools/node-modules

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh
+
+set -eu
+
+GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
+GITHUB_REPOSITORY="${GITHUB_BASE%/*}/node-cache"
+CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
+PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
+
+CACHE_DIR="${XDG_CACHE_HOME-${HOME}/.cache}/cockpit-dev/${GITHUB_REPOSITORY}.git"
+
+message() {
+    [ "${V-}" != 0 ] || printf "  %-8s %s\n" "$1" "$2"
```

Comment:
This is the wrong way around. I want the messages to appear with `make V=1`, right?

Not sure if make exports `V=0` by default, or if it's empty. Probably best to say `[ "${V-}" != 1 ] || printf` and ignore the possibility of `V=2` and similar?

Reply:
Nevermind -- I see the `V_NODE_MODULES_0` bit now, so this is some "inverse" switch.